Don't set Helvetica as default font, it's not available as TrueType
font.  Fixes Bug#477688.

 -- Thomas Weber <thomas.weber.mail@gmail.com>  Sat, 24 May 2008 13:49:51 +0200

diff -urNad octave3.0-3.0.1~/scripts/plot/__go_draw_axes__.m octave3.0-3.0.1/scripts/plot/__go_draw_axes__.m
--- octave3.0-3.0.1~/scripts/plot/__go_draw_axes__.m	2008-04-21 18:00:20.000000000 +0200
+++ octave3.0-3.0.1/scripts/plot/__go_draw_axes__.m	2008-05-25 17:30:08.000000000 +0200
@@ -1601,7 +1601,7 @@
 
 function [f, s, fnt, it, bld] = get_fontname_and_size (t)
   if (isempty (t.fontname))
-    fnt = "Helvetica";
+    fnt = "*";
   else
     fnt = t.fontname;
   endif
@@ -1639,7 +1639,7 @@
   if (strcmp (fld, "string"))
     [f, s, fnt, it, bld] = get_fontname_and_size (obj);
   else
-    f = "Helvetica";
+    f = "*";
     s = 10;
     fnt = f;
     it = false;
diff -urNad octave3.0-3.0.1~/src/graphics.cc octave3.0-3.0.1/src/graphics.cc
--- octave3.0-3.0.1~/src/graphics.cc	2008-04-21 18:00:20.000000000 +0200
+++ octave3.0-3.0.1/src/graphics.cc	2008-05-25 17:30:08.000000000 +0200
@@ -2336,7 +2336,7 @@
     rotation (0),
     horizontalalignment ("left"),
     color (Matrix (1, 3, 0.0)),
-    fontname ("Helvetica"),
+    fontname ("*"),
     fontsize (10),
     fontangle (radio_values ("{normal}|italic|oblique")),
     fontweight (radio_values ("{normal}|bold|demi|light")),
@@ -2470,7 +2470,7 @@
   m["rotation"] = 0;
   m["horizontalalignment"] = "left";
   m["color"] = Matrix (1, 3, 1.0);
-  m["fontname"] = "Helvetica";
+  m["fontname"] = "*";
   m["fontsize"] = 10;
   m["fontangle"] = radio_property (radio_values ("{normal}|italic|oblique"));
   m["fontweight"] = radio_property (radio_values ("{normal}|bold|demi|light"));
